### DISCUSSION TITLE
Transorm checkout into exclusive checkout if it's the only checkout

### DISCUSSION THREAD

**Discussion by Martmists-GH** [NationalSecurityAgency/ghidra#5365] [Ideas]
Currently to obtain an exclusive checkout when I'm the only person with a given file checked out, I have to undo the checkout and then check out again to request exclusive, which leads to having to wait several hours for the file to be redownloaded, despite already having the files in my Ghidra instance. It would be great if this could be changed to force an exclusive checkout from an existing checkout, given no other users have the file checked out.

**Comment by ryanmkurtz** [comment]
Created issue #5368
